package.json

```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest --coverage",
+    "test": "jest --coverage --colors",
```

Comment:
@Bakley good work. Now have this command as `"test": "npm run lint src/**/*.{js,jsx} *.js && jest --coverage --colors"` so that tests will fail when you have linting errors

Reply:
Okay, thanks @koechkevin added the command